internal/server/v2/facet/golden/series_facet/series_facet.json

```diff
@@ -4,205 +4,164 @@
       "by_entity": {
         "Earth": {
           "ordered_facets": [
+            {
+              "facet_id": "3981252704",
```

Comment:
why is there new data here?

Reply:
originally, series facet was only getting facets from one of the BT versions (which was a bug)